packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -513,6 +517,32 @@ class Treemap extends StatelessWidget {
       },
     );
   }
+
+  void _onTapDown(
+    TapDownDetails details,
+    List<PositionedCell> positionedCells,
+  ) {
+    final x = details.localPosition.dx;
+    final y = details.localPosition.dy;
+    // Define target point as rectangle with width and height as 0.01 to
```

Comment:
it is more standard to just intersect a point with the rectangle rather than giving the mouse a "width"

Reply:
Done

Reply:
delete the comment now that it is obsolete